www/book.css

```diff
@@ -30,7 +49,7 @@ header, h1, strong {
     color:black;
 }
 }
-h1 { font-size: 105%; margin: 1.5em 0 0; text-align: left; }
+h1 { font-size: 105%; font-weight: bold; margin: 1.5em 0 0; text-align: left; }
```

Comment:
Why this change?

Reply:
Ah—our browser's default style sheet doesn't bold `h1` tags by default, figured this was safe to add since all other browsers do.